bin/mdsctl

```diff
@@ -333,7 +337,13 @@ templateDockerCompose() {
 
 # TODO: DRY
 templateValues() {
-  suffix=$(getBranchSuffix)
+  branchSuffix=$(getBranchSuffix)
+  if [ "$branchSuffix" = "-master" ]; then
```

Comment:
Wouldn't including the commit hash make sense for `master` as well or are we assuming the version number will be sufficiently descriptive?

Reply:
@invertigo Can probably weigh in more, but for anything in `master` the version number should always match up with one commit. This isn't necessarily the case on other branches (as we make changes without bumping version on each merge to develop)